proto/account.proto

```diff
@@ -3,31 +3,46 @@ package account;
 
 import "digest.proto";
 
+// An account ID.
 message AccountId {
     // A miden account is defined with a little bit of proof-of-work, the id itself is defined as
     // the first word of a hash digest. For this reason account ids can be considered as random
     // values, because of that the encoding below uses fixed 64 bits, instead of zig-zag encoding.
     fixed64 id = 1;
 }
 
+// A summary of an account.
 message AccountSummary {
```

Comment:
"A summary of an account" comment is not very helpful here. I would expand it to a couple of sentences which describe what it is and what it is used for.